others/sko/css/css1.html

```diff
@@ -0,0 +1,20 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style1.css" />
+  </head>
+  <body>
+    <ul class="solarsystem">
+      <li class="pluto"><a href="#pluto"><span>pluto</span></a></li>
```

Comment:
Since we are not adding links, you can remove all the `<a>` tags, such as `<a href="#pluto">`